tests/module_1/test_meteo_api.py

```diff
@@ -1,7 +1,60 @@
 """ This is a dummy example to show how to import code from src/ for testing"""
+import unittest
+from unittest.mock import patch
+import pandas as pd
+from src.module_1.module_1_meteo_api import get_data_meteo_api, process_data
 
-from src.module_1.module_1_meteo_api import main
+#def test_main():
+ #   raise NotImplementedError
 
+class TestWeatherAPI(unittest.TestCase):
```

Comment:
I recomend you to take a look at [pytest](https://docs.pytest.org/en/stable/) framework. Personally I like it more thatn unittest directly